doc/release/v2_3_72.md

```diff
@@ -26,6 +26,7 @@ Important Changes
   `yarp::os::Network` is initialized will abort the program in order to avoid
   undefined behaviour of uninitialized `Time` object
 * `Contactable::open(void)` was deprecated. Use `open("...")` instead.
+* Dropped `YARP1` support.
```

Comment:
Please, move this above, in the generic part of the "Important Changes"  section